package.json

```diff
@@ -28,7 +28,7 @@
   },
   "dependencies": {
     "bindings": "^1.5.0",
-    "node-addon-api": "^4.3.0",
+    "node-addon-api": "^6.1.0",
```

Comment:
Just checking, this change does not require semver major?

Reply:
Do would be semver minor as doesn't affect the public API.